src/components/formik/ActionBlock.jsx

```diff
@@ -23,7 +23,7 @@ const ActionBlock = ({
   const isSubmitting = isFormSubmitting ?? isFormikSubmitting;
 
   return (
-    <div className={classnames(["neeto-ui-action-block__wrapper", className])}>
+    <div className={classnames(["flex items-center gap-x-2", className])}>
```

Comment:
@vijay2136 _a we do not use TailwindCSS within neetoUI. Please replace them with the corresponding [utility classes](https://github.com/bigbinary/neeto-ui/blob/main/src/styles/utilities/_utilities.scss) or SCSS.